client/lib/application/src/application-list/application-list.hbs

```diff
@@ -3,6 +3,19 @@
     <a href="/application/create" class="btn btn-default">
     <i class="fa fa-plus"></i> Create Application</a>
 </div>
+<form>
+    <input
```

Comment:
A `label` would be nice, telling the user what to do with the `input` (see placeholder section at [MDN: input](https://developer.mozilla.org/en/docs/Web/HTML/Element/Input)–a `placeholder` is no replacement for a `label`).